domains/ronan696.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ronan696",
+        "email": "ronan696@gmail.com"
+    },
+    "record": {
+        "CNAME": "ronan696.github.io"
```

Comment:
404 GitHub Page.

Reply:
@DEV-DIBSTER apologies. Corrected the repo name. Should be accessible now. Thanks!